pkg/binary/envoy/debug/node_test.go

```diff
@@ -34,7 +34,8 @@ func Test_debugging_outputs(t *testing.T) {
 		defer os.RemoveAll(r.DebugStore())
 		envoytest.RunKill(r, filepath.Join("testdata", "null.yaml"), time.Second*10)
 
-		files := [...]string{"node/ps.txt", "node/network_interface.json", "node/connections.json"}
+		files := [...]string{"node/ps.txt", "node/network_interface.json", "node/connections.json", "node/iostats.json"}
```

Comment:
```suggestion
		files := []string{"node/ps.txt", "node/network_interface.json", "node/connections.json", "node/iostats.json"}
```

I don't know if I've seen this `[...]` syntax before; either way it's not needed, let's just remove